### PR TITLE
feat: Add option to hide top bar on toggle

### PR DESCRIPTION
**Pull request by sawhney17**
Allows the top bar to be hidden on toggle for a more immersive focus experience.

Screen Shot 2021-12-16 at 10 00 21 AM

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -59,6 +59,28 @@ const main = (inFocusMode = false) => {
                     }
                    `)
                 }
+
+                if(logseq.settings.hide_topbar) {
+                    console.log('top bar', logseq.settings.hide_topbar);
```
Comment (sethfair) [inline src/main.js]: remove debug code

File: src/main.js [hunk 2]
```diff
@@ -59,6 +59,28 @@ const main = (inFocusMode = false) => {
...
+                    logseq.provideStyle(`
+                    div#head:hover {
+                        opacity: 1; 
+                      }
+                    div#head {
+                        opacity: 0;  
+                    }
+                   `)
+                } else {
+                    console.log('top bar shown', logseq.settings.hide_topbar);
```
Comment (sethfair) [inline src/main.js]: remove console.log

File: src/main.js [hunk 3]
```diff
@@ -82,6 +104,18 @@ const main = (inFocusMode = false) => {
                     }
                    `)
                 }
+                
+                if(logseq.settings.hide_topbar) {
+                    console.log('unfocus, top bar shown', logseq.settings.hide_topbar);
```
Comment (sethfair) [inline src/main.js]: remove console.log

File: src/App.js [hunk 4]
```diff
@@ -31,6 +31,7 @@ const App = () => {
                 <OptionCheck checked={settings['go_fullscreen']} setSettings={setSettings} name="go_fullscreen" label="Fullscreen Mode"/>
                 <OptionCheck checked={settings['hide_properties']} setSettings={setSettings} name="hide_properties" label="Page Properties"/>
                 <OptionCheck checked={settings['line_highlight']} setSettings={setSettings} name="line_highlight" label="Line Highlight"/>
+                <OptionCheck checked={settings['hide_topbar']} setSettings={setSettings} name="hide_topbar" label="Hide Topbar"/>
```
Comment (sethfair) [inline src/App.js:34]: add "Hide Topbar" to the readme and update the screenshot.

**Review by sethfair — CHANGES_REQUESTED** [review]
Please address comments prior to merging. Thank you for your submission!

- [x] Update readme and screenshot
- [x] Remove console.log
- [x] Bar should reappear on exit of focus mode

File: src/main.js (hunk 3 above)
Comment (sethfair) [reply src/main.js]: I am not seeing the top bar re-appear on unfocus when the left sidebar remains.  It only hides and shows if the left sidebar remains.

Comment (sawhney17) [reply src/main.js]: Fixed the issue

**Comment by sawhney17** [comment]
@sethfair I've implemented all the changes and fixed the issue with reappearing now.

**Pull request merged**